### PR TITLE
Initial checkin

### PR DESCRIPTION
**Pull request by jayfurmanek**
Add the Tensorflow Hub recipe to Open-CE

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```
Comment (bnemanich) [inline recipe/build.sh]: The date should be updated.

File: recipe/build.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -vex
+
+bazel ${BAZEL_OPTS} build tensorflow_hub/pip_package:build_pip_package
```
Comment (bnemanich) [inline recipe/build.sh:23]: For the other bazel builds, we have been adding:
```
bazel clean --expunge
bazel shutdown
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - python {{ python }}
+    - numpy {{ numpy }}
+    - six {{ six }}
+    - protobuf {{ protobuf }}
+    - tensorflow-base {{ tensorflow }}
+
+about:
+  home: https://www.tensorflow.org/hub/
+  license: Apache-2.0
+  license_file: LICENSE
```
Comment (bnemanich) [inline recipe/meta.yaml]: Does this file need to be added?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: tensorflow-hub
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/hub.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
```
Comment (npanpaliya) [inline recipe/meta.yaml]: build number to be started from 1

File: recipe/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel {{ bazel }}
+  host:
+    - git >=2.2
+    - python {{ python }}
```
Comment (npanpaliya) [inline recipe/meta.yaml]: We don't pin python version for noarch packages.

File: recipe/build.sh [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+mkdir -p $SRC_DIR/tensorflow_hub_pkg
+bazel-bin/tensorflow_hub/pip_package/build_pip_package $SRC_DIR/tensorflow_hub_pkg
+
+ls -l $SRC_DIR/tensorflow_hub_pkg
+
+# install using pip from the whl file
+pip install --no-deps $SRC_DIR/tensorflow_hub_pkg/*.whl
+
+echo "PREFIX: $PREFIX"
+echo "RECIPE_DIR: $RECIPE_DIR"
```
Comment (npanpaliya) [inline recipe/build.sh:35]: Same as Brad's comment above. We add bazel clean and shutdown at the end too.

File: recipe/meta.yaml (hunk 3 above)
Comment (npanpaliya) [reply recipe/meta.yaml]: I see a LICENSE file in the root directory. @jayfurmanek - Are we adding Apache 2.0 License file for all the recipes?

**Review by bnemanich — APPROVED** [review]

**Pull request merged**